tracing/tracingtest/mocktracer.go

```diff
@@ -0,0 +1,41 @@
+package tracingtest
+
+import (
+	"sync/atomic"
+	"time"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/mocktracer"
+)
+
+type MockTracer struct {
+	*mocktracer.MockTracer
```

Comment:
Do we need this field to be exported?

Reply:
Actually, this was a good idea because we need to explicitly code every method of Tracer interface. https://pkg.go.dev/github.com/opentracing/opentracing-go#Tracer

I have defined couple more of them, it's not a big deal at all, but it makes the calls much more understandable: we know which exactly functions are called by mock tests.
https://github.com/zalando/skipper/pull/3322/files#diff-3f5e5fd69ecb51b0c383da632e30fe4bd12ffb855d983235b063b70d0a17ac00R47-R53